homeassistant/components/webostv/media_player.py

```diff
@@ -241,6 +241,19 @@ def _update_states(self) -> None:
             name=self._device_name,
         )
 
+        if (
```

Comment:
```suggestion
        self._attr_assumed_state = True
        if (
```
To be on the safe side, It would be better to set it to True so if a source will not support it we fallback to assume state